### PR TITLE
feat(ic-asset-certification): add function to delete all assets

### PR DESCRIPTION
**Pull request by nathanosdev**

### CODE REVIEW

File: .github/workflows/check-commit-messages.yml [hunk 1]
```diff
@@ -26,4 +26,4 @@ jobs:
           pip install -U Commitizen
 
       - name: Check Commit Messages
-        run: cz check --rev-range c8ecbc19b8c4a482e55907d37554d66f2f2f9a8f..HEAD
+        run: cz check --rev-range ac8f90b7c45e13e240d6c01a43d191b99f1aec4a..HEAD
```
Comment (nathanosdev) [inline .github/workflows/check-commit-messages.yml:29]: I accidentally merged the "update changelog" PR with the wrong commit message so now we have to check commit messages from a later commit starting point

File: packages/ic-asset-certification/README.md [hunk 2]
```diff
@@ -750,6 +750,16 @@ use ic_cdk::api::set_certified_data;
 set_certified_data(&asset_router.root_hash());
 ```
 
+It's also possible to delete all assets and their certification in one go:
+
+```rust
+# use ic_asset_certification::{Asset, AssetConfig, AssetFallbackConfig, AssetRouter, AssetRedirectKind, AssetEncoding};
```
Comment (przydatek) [inline packages/ic-asset-certification/README.md:756]: nit: why do we need to use all these objects when the code is using AssetRouter only?

File: packages/ic-asset-certification/src/asset_router.rs [hunk 3]
```diff
@@ -269,6 +269,9 @@ impl<'content> AssetRouter<'content> {
     ///
     /// If no configuration matches an individual asset, the asset will be
     /// served and certified as-is, without headers.
+    ///
+    /// After performing this operation, the canister's certified variable will need to be updated
```
Comment (przydatek) [inline packages/ic-asset-certification/src/asset_router.rs:273]: nit: I'd rephrase to something more direct, e.g. "After this operation one must set canister's certified data (`ic_cdk::set_certified data()`) to the new [root hash](AssetRouter::root_hash) of the tree." 
(also for the other functions below)

File: packages/ic-asset-certification/src/asset_router.rs [hunk 4]
```diff
@@ -2689,6 +2705,25 @@ mod tests {
         ));
     }
 
+    #[rstest]
+    fn test_delete_all_assets() {
+        let mut asset_router = asset_router();
+        asset_router.delete_all_assets();
```
Comment (przydatek) [inline packages/ic-asset-certification/src/asset_router.rs:2711]: nit: maybe before deletion add a check that `/index.html` is present?

**Review by przydatek — DISMISSED** [review]
LGTM, thanks!

File: packages/ic-asset-certification/README.md (hunk 2 above)
Comment (nathanosdev) [reply packages/ic-asset-certification/README.md:756]: Copy pasta error, thanks!

**Review by przydatek — APPROVED** [review]
LGTM, thanks!
